Makefile

```diff
@@ -155,9 +155,28 @@ SOFTFLOAT_FLAGS  = -I $(SOFTFLOAT_INCDIR)
 SOFTFLOAT_LIBS   = $(SOFTFLOAT_LIBDIR)/softfloat.a
 SOFTFLOAT_SPECIALIZE_TYPE = RISCV
 
+UNAME_S := $(shell sh -c 'uname -s 2>/dev/null || echo not')
+
+STATIC_GMP_A :=
+STATIC_B_STATIC :=
```

Comment:
Might be better to use a more descriptive name here that indicates that these are for different OSs. Something like `STATIC_GMP_MACOS` and `STATIC_LINUX`.

Reply:
I had that originally but it felt clearer to have the logic of how the OS affects what actions are performed in one place, if that makes sense. Maybe `STATIC_LINUX`, `STATIC_MAC` are better though.. anyone else have an opinion?

Reply:
What about something like this then if you want to keep the OS logic together? And just leave the existing `GMP_LIBS` definition as it is.

```make
ifneq (,$(STATIC))
    UNAME_S := $(shell sh -c 'uname -s 2>/dev/null || echo not')
    ifeq ($(UNAME_S),Darwin)
        GMP_LIBS := $(shell pkg-config --variable=libdir gmp)/libgmp.a
        C_LIBS_WRAPPED = $(C_LIBS)
    else
        C_LIBS_WRAPPED = -Wl,--push-state,-Bstatic $(C_LIBS) -Wl,--pop-state
    endif
else
    C_LIBS_WRAPPED = $(C_LIBS)
endif
```

Reply:
Updated to that suggestion, thanks!